layouts/default.vue

```diff
@@ -17,13 +18,25 @@ useMeta({
     },
   ],
 })
+
+onMounted(() => {
+  PullToRefresh.init({
```

Comment:
added a package to handle pull to reload because we are not using the "body". Will look into a better way to use the actual body tag soon